services/storehost/storehost_test.go

```diff
@@ -85,10 +100,9 @@ func (s *StoreHostSuite) TestStoreHostTimerQueueWriteWithRead() {
 		var statsLatency = make([]*stats, numExtents)
 		var statsDrift = make([]*stats, numExtents)
 
+		wgAll.Add(numExtents)
+		var readerTimeout int32
 		for x := 0; x < numExtents; x++ {
-
-			wgAll.Add(1)
-
```

Comment:
this is okay .. but just thought i'd let you know, this is a pattern i tend to use .. where the wait-group add immediately precedes the go-func call; and the first thing the go-func does is a 'defer waitgroup-done'.